SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkerRegion.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "LoadBalancing/WorkerRegion.h"
+
+#include "Materials/MaterialInstanceDynamic.h"
+#include "UObject/ConstructorHelpers.h"
+#include "UObject/UObjectGlobals.h"
+
+DEFINE_LOG_CATEGORY(LogWorkerRegion);
+
+const float AWorkerRegion::DEFAULT_WORKER_REGION_HEIGHT = 10;
```

Comment:
Maybe we could expose these as settings on the Spatial Debugger (or elsewhere) that get passed in?  Maybe even through WorldSettings (can we have a SpatialWorldSettings please :) )  At a minimum i'd expect the height will need to be adjusted per-level until we have a solution that projects these colors onto all geometry.

Reply:
I experimented with having a SpatialDebugger UPROPRTY which was editable in game but I failing to make it work so temporarily dropped having that. The same result was easily achieved by just group selecting the regions in the world outliner and changing their height together. I'd rather stick with this and focusing on a more formal way to do this (either via world settings or a SpatialDebugger property) in a followup task?

Reply:
sounds good